lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -273,10 +273,14 @@
     custom_compatibility_version:
         description:
             - "Enables a virtual machine to be customized to its own compatibility version. If
-            `C(custom_compatibility_version)` is set, it overrides the cluster's compatibility version
+            'C(custom_compatibility_version)' is set, it overrides the cluster's compatibility version
             for this particular virtual machine."
         version_added: "2.7"
-
+    host_devices:
+        description:
+            - Single Root I/O Virtualization - technology that allows single device to expose multiple endpoints that can be passed to VMs
+            - host_devices is an list which contain dictinary with name and state of device
```

Comment:
Take a look how for example `cpu_pinning` is documented, and please document it the same way.